src/test/kotlin/nl/info/zac/admin/ZaaktypeConfigurationServiceTest.kt

```diff
@@ -0,0 +1,148 @@
+/*
+ * SPDX-FileCopyrightText: 2025 INFO.nl
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package nl.info.zac.admin
+
+import io.kotest.core.spec.style.BehaviorSpec
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.runs
+import io.mockk.verify
+import jakarta.persistence.EntityManager
+import jakarta.persistence.criteria.CriteriaQuery
+import nl.info.client.zgw.ztc.ZtcClientService
+import nl.info.client.zgw.ztc.model.createZaakType
+import nl.info.zac.admin.model.ZaaktypeConfiguration
+import nl.info.zac.admin.model.createZaaktypeCmmnConfiguration
+import nl.info.zac.flowable.bpmn.model.createZaaktypeBpmnConfiguration
+import java.net.URI
+import java.util.UUID
+
+class ZaaktypeConfigurationServiceTest : BehaviorSpec({
+    val zaaktypeUri = URI("https://example.com/zaaktype/${UUID.randomUUID()}")
+
+    val entityManager = mockk<EntityManager>()
+    val ztcClientService = mockk<ZtcClientService>()
+    val zaaktypeCmmnConfigurationBeheerService = mockk<ZaaktypeCmmnConfigurationBeheerService>()
+    val zaaktypeBpmnConfigurationBeheerService = mockk<ZaaktypeBpmnConfigurationBeheerService>()
+    val zaaktypeConfigurationService = ZaaktypeConfigurationService(
+        entityManager,
+        ztcClientService,
+        zaaktypeCmmnConfigurationBeheerService,
+        zaaktypeBpmnConfigurationBeheerService
+    )
+
+    Given("a concept zaaktype event") {
```

Comment:
Not sure about others but I like to put a `Context` around all tests, just because it makes it easier to see which method we are testing in these tests. E.g. in this case `Context("Updating zaakafhandelparameters") { .. `